### DISCUSSION TITLE
ARIA Not Working with Zotero I am getting error messages

### DISCUSSION THREAD

**Discussion by Foreverclever** [lifan0127/ai-research-assistant#98] [General]
Hi
I am getting the following error messages, I have a chat gpt account and have put in the correct open ai code etc

I need some to please setup my Zotero with Aria plugin so that it all works for chat gpt4, I am not that techy and I am struggling, I need to be able to use chat gpt4 for free with no limitations, I understand it can be done with microsoft pilot, bling etc but don't know how to get the open api key etc?

Does anyone have a working Zotero with Aria plugin that I can download and it will work for me with no issues ?

Error messages

Question show me the papers related to rail freight

{
  "status": 401,
  "headers": {},
  "error": {
    "message": "Incorrect API key provided: YOUR_OPE*******_KEY. You can find your API key at https://platform.openai.com/account/api-keys.",
    "type": "invalid_request_error",
    "param": null,
    "code": "invalid_api_key"
  },
  "code": "invalid_api_key",
  "param": null,
  "type": "invalid_request_error",
  "attemptNumber": 1,
  "retriesLeft": 6,
  "name": "Error",
  "message": "401 Incorrect API key provided: YOUR_OPE*******_KEY. You can find your API key at https://platform.openai.com/account/api-keys.",
  "stack": "@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:830:19\n_APIError@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54432:5\nAuthenticationError<@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54513:11\ngenerate@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54463:14\nmakeStatusError@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:55190:12\nmakeRequest@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:55230:20\n"
}
show me documents

{
  "status": 404,
  "headers": {},
  "error": {
    "message": "The model `gpt-4-0613` does not exist or you do not have access to it.",
    "type": "invalid_request_error",
    "param": null,
    "code": "model_not_found"
  },
  "code": "model_not_found",
  "param": null,
  "type": "invalid_request_error",
  "attemptNumber": 1,
  "retriesLeft": 6,
  "name": "Error",
  "message": "404 The model `gpt-4-0613` does not exist or you do not have access to it.",
  "stack": "@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:830:19\n_APIError@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54432:5\nNotFoundError<@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54525:11\ngenerate@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54469:14\nmakeStatusError@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:55190:12\nmakeRequest@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:55230:20\n"
}
then i get a different error message

show me the papers

{
  "status": 500,
  "headers": {},
  "attemptNumber": 7,
  "retriesLeft": 0,
  "name": "Error",
  "message": "500 status code (no body)",
  "stack": "@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:830:19\n_APIError@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54432:5\n@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:830:19\ngenerate@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:54481:14\nmakeStatusError@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:55190:12\nmakeRequest@resource://gre/modules/addons/XPIProvider.jsm -> jar:file://extensions/aria@apex974.com.xpi!/bootstrap.js -> jar:file://extensions/aria@apex974.com.xpi!/chrome/content/scripts/index.js:55230:20\n"
}

**Comment by pallykkja** [comment]
I tried to help a PHD today with using the GPT-4 and got a similar error message. https://forums.zotero.org/discussion/114736/gpt-4-aria-connection-problem  what can we do?

**Reply by pallykkja** [reply]
Problem solved by buying a seperate extra "top up" on Open AI.
